src/components/data-processing/GenesVsUMIs/CalculationConfig.jsx

```diff
@@ -1,24 +1,63 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import {
   Space,
   InputNumber,
   Form,
   Tooltip,
   Select,
+  Slider,
+  Alert,
+  Radio,
+  Button,
 } from 'antd';
+import { useDispatch } from 'react-redux';
 import { InfoCircleOutlined } from '@ant-design/icons';
+import useUpdateThrottled from 'utils/customHooks/useUpdateThrottled';
+import { runQC } from 'redux/actions/pipeline';
 
 const { Option } = Select;
 
 const GenesVsUMIsConfig = (props) => {
+  const dispatch = useDispatch();
   const {
-    // eslint-disable-next-line no-unused-vars
-    config, disabled, plotType, updateSettings,
+    config, updateSettings, disabled, rerunRequired, experimentId,
   } = props;
+  const [newConfig, handleChange] = useUpdateThrottled(updateSettings, config);
+
+  // if prediction interval is not set - use the passed p-value from the pipeline
+  useEffect(() => {
+    if (!config.predictionInterval) {
+      updateSettings(
+        { predictionInterval: parseFloat(1 - config.regressionTypeSettings[config.regressionType]['p.level']).toFixed(6) },
+        false,
+      );
+    }
+  }, []);
 
   return (
     <>
+      {/* only display info message for datasets which have
+      not rerun the pipeline to see the new interractive plot */}
+      {rerunRequired && (
+        <Space direction='vertical'>
+          <Alert
+            message='A new interactive version of this plot is available! To see it,
+      you need to run data processing again by clicking on the button below.
+      Please note that by doing so, any customisation made to your project will be lost.
+      In this case, we suggest creating a new project with the same data to view the new interactive plot.'
+            type='info'
+            showIcon
+          />
```

Comment:
They can choose to re-run if they want the new interactivity, but if they don't want it or need it they can choose not to.  we could add something to this comment saying that.

Reply:
```suggestion
      {/* only display info message for datasets which have
      not rerun the pipeline to see the new interractive plot */}
      {rerunRequired && (
        <Space direction='vertical'>
          <Alert
            message='A new interactive version of this plot is available! To see it,
      you need to run data processing again by clicking on the button below.
      Please note that by doing so, any customisation made to your project will be lost.
      In this case, we suggest creating a new project with the same data to view the new interactive plot. In case you do not need the interactivity, you can choose not to re-run data processing, and continue using the platform as it is.'
            type='info'
            showIcon
          />
```

Reply:
Those were the instructions in the ticket. We are letting them know there is a new version in the plot.

Reply:
Can we simplify the wording? I think it's too long.

Reply:
> Can we simplify the wording? I think it's too long.

The message was already reviewed and approved by Vicky. I think it's worth to explain this in a proper way to the user, even if it's a bit longer. But if we come up with a shorter version, we can submit it to Vicky again.